pyproject.toml

```diff
@@ -98,6 +98,9 @@ known-first-party = [
   'mesonpy',
 ]
 
+[tool.ruff.format]
+# Use single quotes for strings
```

Comment:
This comment is redundant: it is pretty clear what the purpose of the option is. Please remove it.